<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifierWindow</class>
 <widget class="QMainWindow" name="VerifierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RapidSum Checksum Verifier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="labelFiles">
        <property name="text">
         <string>Files: -/-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxOK">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OK: 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxBad">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Bad: 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxMissing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Missing: 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Checksum</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: transparent;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOperation"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOperation">
   <property name="text">
    <string>File Operation...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
